chain/bitcoind_client.go

```diff
@@ -22,6 +23,13 @@ import (
 	"github.com/lightninglabs/gozmq"
 )
 
+var (
+	// ErrBitcoindClientShuttingDown is an error returned when we attempt
+	// to receive a notification for a specific item and the bitcoind client
+	// is in the middle of shutting down.
+	ErrBitcoindClientShuttingDown = errors.New("client is shutting down")
```

Comment:
Are we able to recover from this error?

Reply:
Nope. The only way this error can be returned is if the caller explicitly called `Stop` on either the `BitcoindClient` or `BitcoindConn`.